test/parallel/test-dgram-exclusive-implicit-bind.js

```diff
@@ -85,7 +83,12 @@ if (cluster.isMaster) {
   return;
 }
 
-var source = dgram.createSocket('udp4');
+const source = dgram.createSocket('udp4');
+var interval;
+
+source.on('close', function() {
+  clearTimeout(interval);
+});
```

Comment:
From a previous comment I'm not sure this was a problem but can't you get rid of this as the interval is already unrefed? At least locally it works for me